src/api/libcellml/variable.h

```diff
@@ -140,28 +140,36 @@ class LIBCELLML_EXPORT Variable: public NamedEntity
 
     /**
      * @brief Get the initial value for this variable.
-     * Get the real number initial value for this variable.
+     * Get the string corresponding to the initial value for this variable.
      * @sa setInitialValue
      */
     std::string getInitialValue() const;
 
     /**
      * @brief Set the interface type for this variable.
+     * Set the interface type for this variable using a string.
+     * @sa getInterfaceType
+     * @param interfaceType The string interface type to set for this variable.
+     */
+    void setInterfaceType(const std::string &interfaceType);
+
+    /**
+     * @brief Set the interface type for this variable.
      * Set the interface type for this variable from the available
-     * options in the INTERFACE_TYPES enum.
+     * options in the InterfaceTypes enum class.
+     *
+     * @overload
      * @sa getInterfaceType
-     * @param interfaceType The interface type to set for this variable.
+     * @param interfaceType The enum interface type to set for this variable.
      */
-    void setInterfaceType(INTERFACE_TYPES interfaceType);
+    void setInterfaceType(InterfaceTypes interfaceType);
```

Comment:
OK I'll change this up in the next round of tidy-up (this is a bit far-reaching to sneak into another changeset). It's implied in the examples in the [Google C++ style guide](https://google.github.io/styleguide/cppguide.html#Enumerator_Names) that they use plural `enum` names but not explicitly stated. I agree that singular reads better :)